sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/InputRedundantReturnCheckNestedMethods.java

```diff
@@ -0,0 +1,41 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+public class InputRedundantReturnCheckNestedMethods {
+    public void method() {
+        try {
+            if (true) {
+                new Nested() {
+                    @Override
+                    public boolean accept(String s) {
+                        return true; // violation
```

Comment:
why violation is here ?

redundant return is only "return;". Am i miss smth ?

Reply:
Done.